contracts/contracts/strategies/balancer/BalancerMetaPoolStrategy.sol

```diff
@@ -0,0 +1,301 @@
+pragma solidity ^0.8.0;
+
+/**
+ * @title OETH Balancer MetaStablePool Strategy
+ * @author Origin Protocol Inc
+ */
+import { SafeERC20 } from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import { BaseAuraStrategy } from "./BaseAuraStrategy.sol";
+import { IBalancerVault } from "../../interfaces/balancer/IBalancerVault.sol";
+import { IRateProvider } from "../../interfaces/balancer/IRateProvider.sol";
+import { IMetaStablePool } from "../../interfaces/balancer/IMetaStablePool.sol";
+import { IERC20 } from "../../utils/InitializableAbstractStrategy.sol";
+import { StableMath } from "../../utils/StableMath.sol";
+
+import "hardhat/console.sol";
+
+contract BalancerMetaPoolStrategy is BaseAuraStrategy {
+    using SafeERC20 for IERC20;
+    using StableMath for uint256;
+
+    address internal immutable stETH =
+        0xae7ab96520DE3A18E5e111B5EaAb095312D7fE84;
+    address internal immutable wstETH =
+        0x7f39C581F595B53c5cb19bD0b3f8dA6c935E2Ca0;
+    address internal immutable frxETH =
+        0x5E8422345238F34275888049021821E8E08CAa1f;
+    address internal immutable sfrxETH =
+        0xac3E018457B222d93114458476f3E3416Abbe38F;
+
+    function getRateProviderRate(address _asset)
```

Comment:
Where is `getRateProviderRate` used?

Reply:
Used to be used in `getBPTExpected` but when refactored out of it I hadn't noticed that the function isn't used anymore